perllib/FixMyStreet/Map/OSM.pm

```diff
@@ -77,11 +77,15 @@ sub generate_map_data {
         $zoomOffset = 0;
     }
 
-    # Adjust zoom level dependent upon population density
-    my $dist = $data->{distance}
-        || FixMyStreet::Gaze::get_radius_containing_population( $params{latitude}, $params{longitude} );
-    my $default_zoom = $data->{cobrand}->default_map_zoom() || ($numZoomLevels - 4);
-    $default_zoom = $numZoomLevels - 3 if $dist < 10;
+    # Adjust zoom level dependent upon population density if cobrand hasn't
+    # specified a default zoom.
+    my $cobrand_default_zoom = $data->{cobrand}->default_map_zoom();
+    my $default_zoom = $cobrand_default_zoom || ($numZoomLevels - 4);
+    if (undef $cobrand_default_zoom) {
```

Comment:
I assume this should be `!defined`; undef always returns undef.

Reply:
I need this for another branch, hence me spotting this, sorry for not doing so before! Would this be slightly clearer?
```
my $default_zoom;
if (my $cobrand_default_zoom = $data->{cobrand}->default_map_zoom) {
    $default_zoom = $cobrand_default_zoom;
} else {
    my $dist = $data->{distance}
        || FixMyStreet::Gaze::get_radius_containing_population( $params{latitude}, $params{longitude} );
    $default_zoom = $dist < 10 ? $numZoomLevels - 3 : $numZoomLevels - 4;
}
```

Reply:
Nice catch, whoops! Yes, your suggestion is clearer 👍